jiant/tasks/lib/templates/shared.py

```diff
@@ -174,6 +214,21 @@ def create_generic_data_row_from_tokens_and_segments(
     feat_spec: FeaturizationSpec,
     data_row_class,
 ):
+    """Creates an input set and wraps input set into a DataRow class.
```

Comment:
Consider using `InputSet` since it's an actual class.

Reply:
Thanks, addressed w/ 33c7f2b.